raiden/transfer/mediated_transfer/events.py

```diff
@@ -305,10 +305,10 @@ def from_dict(cls, data: typing.Dict[str, typing.Any]) -> 'SendBalanceProof':
         restored = cls(
             recipient=to_canonical_address(data['recipient']),
             channel_identifier=typing.ChannelID(int(data['channel_identifier'])),
-            message_identifier=int(data['message_identifier']),
-            payment_identifier=int(data['payment_identifier']),
+            message_identifier=typing.MessageID(int(data['message_identifier'])),
```

Comment:
Both changes to ID type should be fine since in the runtime they are all ints anyway.
So I guess doing something like: `typing.MessageID(data['message_identifier'])` would also work?